### ISSUE TITLE
Typescript warnings for @glimmer/wire-format typings

### ISSUE THREAD

**Issue by rwjblue** [glimmerjs/glimmer-vm#460]
When running:

```
ember new glimmer-test-app -b @glimmer/blueprint
cd glimmer-test-app
ember s
```

I get the following console output:

```
/node_modules/@glimmer/wire-format/dist/types/index.d.ts(141,22): Duplicate identifier 'SerializedTemplateBlockJSON'.
/node_modules/@glimmer/wire-format/dist/types/index.d.ts(143,21): Duplicate identifier 'SerializedTemplateBlockJSON'.
/node_modules/@glimmer/wire-format/dist/types/index.d.ts(155,22): Duplicate identifier 'TemplateJavascript'.
/node_modules/@glimmer/wire-format/dist/types/index.d.ts(157,21): Duplicate identifier 'TemplateJavascript'.
```

---

I'm unsure why these duplicate identifiers exist, any ideas?

**Comment by tomdale** [comment]
https://github.com/glimmerjs/glimmer-vm/pull/461

**Comment by simi** [comment]
I think this can be closed since #461 is merged.

**Comment by rwjblue** [comment]
Thanks!
